src/Engine/ProtoCore/Namespace/ElementRewriter.cs

```diff
@@ -11,8 +11,15 @@
 namespace ProtoCore.Namespace
 {
 
-    public static class ElementRewriter
+    public class ElementRewriter
```

Comment:
Is this class more for extensibility of the rewriter?

As opposed to just one function LookupResolvedNameAndRewriteAst(...)